aws/resource_aws_gamelift_fleet.go

```diff
@@ -70,6 +70,19 @@ func resourceAwsGameliftFleet() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringLenBetween(1, 1024),
 			},
+			"locations": {
```

Comment:
I suggest renaming this argument to `location_configuration` to match the underlying AWS API.